install/jaxr/docs/JAXRSpecAssertions.html

```diff
@@ -843,7 +843,7 @@ <h2> Jakarta Registries 1.0 - 1.0<br>
 </font></TD><TD valign="center" align="center"><font size="1PT">true</font></TD><TD valign="center" align="center"><font size="1PT">technology</font></TD><TD valign="center" align="center"><font size="1PT">active</font></TD><TD valign="center" align="center"><font size="1PT">false</font></TD>
 </TR>
```

Comment:
I think removing install/jaxr is expected, I've never pruned a TCK test before but that is what I think is right.  Good that we have others that know for sure.